internal/compiler-bridge/src/main/scala/xsbt/CallbackGlobal.scala

```diff
@@ -225,13 +226,13 @@ sealed class ZincCompiler(settings: Settings, dreporter: DelegatingReporter, out
       if (sym.isRoot || sym.isRootPackage || sym == NoSymbol || sym.owner.isEffectiveRoot) {
         val capacity = size + nSize
         b = new java.lang.StringBuffer(capacity)
-        b.append(chrs, symName.start, nSize)
+        b.append(chrs, encodedName.start, nSize)
```

Comment:
[error] /home/travis/build/sbt/zinc/internal/compiler-bridge/src/main/scala/xsbt/CallbackGlobal.scala:229:24: type mismatch;
[error]  found   : encodedName.type (with underlying type String)
[error]  required: ?{def start: ?}
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method stringToTermName in trait Names of type (s: String)ZincCompiler.this.TermName
[error]  and method stringToTypeName in trait Names of type (s: String)ZincCompiler.this.TypeName
[error]  are possible conversion functions from encodedName.type to ?{def start: ?}
[error]         b.append(chrs, encodedName.start, nSize)
[error]                        ^